Partial Match fixes (#13)
helibproject/HElib: src/CMakeLists.txt

endif (BUILD_SHARED)

if (ENABLE_TEST)

>   # Add tests targets (so thy will be built) and also all tests to cmake.

Author: ```suggestion
  # Add test targets (so they will be built) and all tests to cmake.
```
